content/en/digital_certificate.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely.
+
+Certificates authenticate the user through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
+
+## Problem it addresses
```

Comment:
I think this needs to be reworked. While authenticating a user is one possible use for certificates there are many others including the much more common use of providing the underlying identity and encryption mechanism for TLS.

This entire section needs to be rewritten in a way that is more broadly applicable. 

That being said this is a great effort and I really appreciate your work.